### ISSUE TITLE
cilium: -n should either take full address or specify prefix.

### ISSUE THREAD

**Issue by borkmann** [cilium/cilium#960]
Option -n says:
```
# cilium-agent --help
[...]
  -n, --node-address string           IPv6 address of node, must be in correct format
[...]
```
Passing the address of my node results in an error when starting the agent:
`
Jun 12 17:25:27 linux-2.home cilium-agent[8639]: level=fatal msg="Unable to parse node address: Endpoint ID set in IPv6 node address"`

Passing /112 prefix works however. Either we should specify this explicitly in the log, or better just take the address and mask out the bits internally. Perhaps latter might be easier to use.

**Comment by tgraf** [comment]
Likely addressed by #969, to be confirmed.

**Comment by tgraf** [comment]
Fixed in #1020
